source/Hardpoint.cpp

```diff
@@ -133,6 +133,21 @@ bool Hardpoint::IsTurret() const
 
 
 
+// Find out if this is a pylon hardpoint (whether or not it has a projectile installed).
```

Comment:
```suggestion
// Find out if this is a pylon hardpoint (whether or not it has a missile weapon installed).
```

Reply:
Might be better to just say "weapon" instead, since while it will *probably* be an ammunition weapon, it doesn't actually have to be. Could, for example, be something that uses energy to create a ball of guided energy that drops off the side of the ship and then engages in missile-like tracking behavior.

Reply:
Alright.